pkg/publicapi/constants.go

```diff
@@ -1,4 +0,0 @@
-package publicapi
-
-const LegacyAPIPrefix = ""
-const V1APIPrefix = "/api/v1"
```

Comment:
constants this way made it more difficult to know where paths are registered as the path is broken into different constants (i.e. `V1APIPrefix` + `ComputeSuffix`) which is more difficult to search for in the code compared to `/api/v1/compute`. Also different endpoints can have different versions. 

I can bring back constants under each endpoint that has the full path as in `ComputePath = "/api/v1/compute"`, but won't use this constant in our tests as changing the path should make it clear that tests should be updated since this will also break existing clients.

Thoughts?

Reply:
Maybe do `ComputePathV1 = "/api/v1/compute"`? Skipping usage in tests sounds good as a prompt to update them when things change.